src/generate.rs

```diff
@@ -12,29 +12,26 @@ pub enum ErrorModel {
     Gap,
     /// Delete a region of size e*n and insert a region of size e*n.
     Move,
-    /// Takes a region of size e*n/2 and insert it
+    /// Takes a region of size e*n and insert it
     Insert,
     /// Takes a region of size e*n/2 and inserts it twice in a row next to
     /// each other
     Doubleinsert,
-    /// Construct the sequence of e*n repeating subsequences B and mutates each
-    /// of them randomly
-    RepeatedPattern,
     /// Construct the sequence of e*n repeating subsequences for sequence A
-    /// and adds sequence_length*error_rate mutations for sequence B
+    /// and adds e*n mutations for sequence B
     Repeat,
     /// Construct the sequence of e*n repeating subsequences for sequence and adds
-    /// sequence_length*error_rate mutations for sequence A, and then adds
-    /// sequence_length*error_rate mutations for sequence B
+    /// e*n mutations for sequence A, and then adds
+    /// e*n mutations for sequence B
     MutatedRepeat,
     /// Construct the sequence of e*n repeating subsequences for sequence and adds
-    /// sequence_length*error_rate mutations for sequences A and B individually
+    /// e*n mutations for sequences A and B individually
     DoubleMutatedRepeat,
 }
 
 #[derive(Parser, Clone)]
 pub struct GenerateArgs {
-    /// The number of sequence pairs to generate
+    /// The number of repetitive patterns from which A is constracted
```

Comment:
`cnt` should only be used for the number of sequence pairs to generate, so the previous was good.